packages/portal-proto/src/features/repositoryApp/FilesTable.tsx

```diff
@@ -360,7 +360,8 @@ const FilesTables: React.FC = () => {
       endpoint: "files",
       method: "POST",
       params: {
-        filters: buildCohortGqlOperator(allFilters) ?? {},
+        case_filters: buildCohortGqlOperator(cohortFilters) ?? {},
+        filters: buildCohortGqlOperator(repositoryFilters) ?? {},
```

Comment:
Do we need both?

Reply:
Yes. `case_filters` are for cohorts and `filters` are local filters. This will be needed when gene/ssm filters are part of the cohort (and not represented with a case set).